<ui version="4.0" >
 <class>TicketOrderDlg</class>
 <widget class="QDialog" name="TicketOrderDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Ticket Order #1</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="4" column="0" colspan="6" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>361</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="5" >
    <widget class="QLabel" name="amountLabel" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>$</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>&amp;Quantity:</string>
     </property>
     <property name="buddy" >
      <cstring>quantitySpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>&amp;Price:</string>
     </property>
     <property name="buddy" >
      <cstring>priceSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Amount</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QDoubleSpinBox" name="priceSpinBox" >
     <property name="alignment" >
      <set>Qt::AlignRight</set>
     </property>
     <property name="prefix" >
      <string>$ </string>
     </property>
     <property name="maximum" >
      <double>5000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QSpinBox" name="quantitySpinBox" >
     <property name="alignment" >
      <set>Qt::AlignRight</set>
     </property>
     <property name="maximum" >
      <number>50</number>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="value" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>81</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&amp;When:</string>
     </property>
     <property name="buddy" >
      <cstring>whenDateTimeEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QDateTimeEdit" name="whenDateTimeEdit" />
   </item>
   <item row="0" column="1" colspan="5" >
    <widget class="QLineEdit" name="customerLineEdit" />
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&amp;Customer:</string>
     </property>
     <property name="buddy" >
      <cstring>customerLineEdit</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>customerLineEdit</tabstop>
  <tabstop>whenDateTimeEdit</tabstop>
  <tabstop>priceSpinBox</tabstop>
  <tabstop>quantitySpinBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TicketOrderDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>222</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel" >
     <x>119</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TicketOrderDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>321</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel" >
     <x>142</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
